discord/ext/commands/converter.py

```diff
@@ -823,28 +828,55 @@ def repl(obj):
         # Completely ensure no mentions escape:
         return discord.utils.escape_mentions(result)
 
-class _Greedy:
-    __slots__ = ('converter',)
 
-    def __init__(self, *, converter=None):
+GreedyBase = list if sys.version_info >= (3, 9) else List
```

Comment:
I don't really know how I feel about guards like these.

Right now we're targeting 3.8 and in 3.9 we don't have to use `List`. In the future, i.e. like 4 years from now `List` will probably break. By that time (assuming Discord and this library survives that long) we'd have moved on from 3.8 and the minimum Python version would be greater than 3.9. So I'm not sure if littering the code with these guards accomplishes much.